src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -89,19 +90,34 @@ async def exec_line_async(
         inputs: Mapping[str, Any],
         index: Optional[int] = None,
         run_id: Optional[str] = None,
+        allow_generator_output=False,
     ) -> LineResult:
         start_time = datetime.utcnow()
         # call execution api to get line results
         url = self.api_endpoint + "/execution"
         payload = {"run_id": run_id, "line_number": index, "inputs": inputs}
-        async with httpx.AsyncClient() as client:
-            response = await client.post(url, json=payload, timeout=LINE_TIMEOUT_SEC)
-        # process the response
-        result = self._process_http_response(response)
-        if response.status_code != 200:
-            run_info = FlowRunInfo.create_with_error(start_time, inputs, index, run_id, result)
-            return LineResult(output={}, aggregation_inputs={}, run_info=run_info, node_run_infos={})
-        return LineResult.deserialize(result)
+        headers = {"Accept": "text/event-stream"} if allow_generator_output else None
+
+        if allow_generator_output:
```

Comment:
We should keep the return result consistent with python exec_line.
Example result should be something like this:
```
LineResult(
outputs={
"normal_output": "xxx",
"streaming_output": Generator of the string,
}
run_info=FlowRunInfo(xxx),
node_run_infos={},
)
```